pytorch/buildspec-1-5-eia.yml

```diff
@@ -29,9 +29,9 @@ images:
     build: &PYTORCH_CPU_INFERENCE_PY3 false
     image_size_baseline: 6318
     device_type: &DEVICE_TYPE cpu
-    python_version: &DOCKER_PYTHON_VERSION py3
-    tag_python_version: &TAG_PYTHON_VERSION py36
-    os_version: &OS_VERSION ubuntu16.04
+    python_version: &DOCKER_PYTHON_VERSION py38
```

Comment:
The issue with the standard labels test appears to be arising from the fact that "device_type" is set to "cpu" instead of eia. Can we change device_type to "eia", and then in "tag" replace *DEVICE_TYPE with "cpu"

Reply:
Modified